### DISCUSSION TITLE
Virustotal results on v1.3.13 Windows Release

### DISCUSSION THREAD

**Discussion by tuanvoffice** [NUWCDIVNPT/stig-manager#1141] [Q&A]
Virustotal detects "Trojan.Agent.Win32.2853154" on v1.3.13 Windows Release.
Is this a false positive on this version of stig-manager? 

https://www.virustotal.com/gui/file/9ea5528b86843bd0f13b84d3f3b32f4fa7cb530b161c5d25381f63b25c3d5f2c/detection

**Comment by cd-rite** [comment]
Hi @tuanvoffice Yes, that would be a false positive. The Rule info for the flagged behavior at that link says:
`falsepositives:
    - Custom applications use renamed binaries adding slight change to binary name. Typically this is easy to spot and add to whitelist`

And that is likely why it was flagged. We rename our application as part of the build process.  
If you are still concerned, you might consider using our Container Image,  since that runs our source code directly and could be more easily inspected, instead of being wrapped up in an executable.
